202305_dc0/common.toml

```diff
@@ -0,0 +1,9 @@
+lmax_over_modeling_nside = 2.5
+healpix = true
+car = false
+output_folder = "output_lsq/{nside}/{tag}/0000"
+output_filename_template= "cmbs4_{tag}_uKCMB_{telescope}-{band}_nside{nside}_0000.fits"
```

Comment:
@jdborrill as for file naming, I don't think we want to implement the Data Schema naming here.

Reply:
Why not?

Reply:
Data Schema naming is based on component bits, we cannot use it here as we have too many components.
Also the `mapXX` filename component won't be as useful, we only have 1 type of map.
So not much is left of the file naming schema.

Reply:
Fair enough. What is "tag" here?

Reply:
is the name of the component, for example `dust_medium`

Reply:
Would it be worth using the internal PySM component naming? "high", "medium" and "low" complexity will (i) change with time, and (ii) only apply to the collection of components that define a total model. So I'd only expect us to use them (with version numbers) for the combined maps.

Reply:
so, here are the components with both names:

```
ame_high.toml:pysm_components_string = "a2"
ame_medium.toml:pysm_components_string = "a1"
cib.toml:pysm_components_string = "cib1"
cmb_lensed.toml:pysm_components_string = "c4"
cmb_unlensed.toml:pysm_components_string = "c3"
co_low.toml:pysm_components_string = "co1"
co_medium.toml:pysm_components_string = "co3"
dust_high.toml:pysm_components_string = "d12"
dust_low.toml:pysm_components_string = "d9"
dust_medium.toml:pysm_components_string = "d10"
freefree_medium.toml:pysm_components_string = "f1"
ksz.toml:pysm_components_string = "ksz1"
radio.toml:pysm_components_string = "rg1"
synchrotron_high.toml:pysm_components_string = "s7"
synchrotron_low.toml:pysm_components_string = "s4"
synchrotron_medium.toml:pysm_components_string = "s5"
tsz.toml:pysm_components_string = "tsz1"
```

Maybe combine the 2, so do `dust_d12`, `dust_d9`? So some of them which are obscure, like `c3`, or `rg1` , become `cmb_c3` and `radio_rg1`.

Reply:
Sounds good.